README.md

```diff
@@ -174,6 +174,7 @@ GWT Boot Awesome Library List (Gwit a LiLi) - Collection of JavaScript Libraries
 
 - [gwt-jackson](https://github.com/nmorel/gwt-jackson) - Jackson library
 - [jackson-xml](https://github.com/treblereel/jackson-xml) - Jackson library
+- [domino-jackson](https://github.com/DominoKit/domino-jackson) - Jackson library
```

Comment:
Shouldnt this be in the top of the list since it starts with the letter `D` and it uses annotation processors vs `gwt-jackson` using generators.